types/sinch-rtc/index.d.ts

```diff
@@ -0,0 +1,140 @@
+// Type definitions for non-npm package sinch-rtc-browser 2.31
+// Project: https://developers.sinch.com/docs/in-app-calling/js-cloud/
+// Definitions by: Johan Aschan <https://github.com/sinch-johana>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const Sinch: SinchRtc.Sinch;
+
+declare namespace SinchRtc {
+    interface Sinch {
+        getSinchClientBuilder(): SinchClientBuilder;
+        version(): string;
+    }
+
+    interface SinchClientBuilder {
+        userId(userId: string): SinchClientBuilder;
+        applicationKey(applicationKey: string): SinchClientBuilder;
+        environmentHost(environmentHost: string): SinchClientBuilder;
+        callerIdentifier(callerIdentifier: string): SinchClientBuilder;
+        build(): SinchClient;
+    }
+
+    interface SinchClient {
+        start(): Promise<void>;
+        isStarted(): boolean;
+        pushNotificationDisplayName?: string;
+        addListener(sinchClientListener: SinchClientListener): void;
+        removeListener(sinchClientListener: SinchClientListener): void;
+        setSupportManagedPush(serviceWorker?: string): Promise<void>;
+        terminate(): void;
+        callClient: CallClient;
+    }
+
+    interface CallClient {
+        callUser(toUserId: string, headers?: Record<string, string>): Promise<Call>;
+        callUserVideo(toUserId: string, headers?: Record<string, string>): Promise<Call>;
+        callPhoneNumber(phoneNumber: string, headers?: Record<string, string>): Promise<Call>;
+        callSip(sipIdentity: string, headers?: Record<string, string>): Promise<Call>;
+        callConference(conferenceId: string, headers?: Record<string, string>): Promise<Call>;
+        getCall(callId: string): Call | undefined;
+    }
+
+    interface Call {
+        addListener(listener: CallListener): void;
+        removeListener(listener: CallListener): void;
+        hangup(): void;
+        answer(): void;
+        mute(): void;
+        unmute(): void;
+        pauseVideo(): void;
+        resumeVideo(): void;
+        sendDtmf(keys: string): void;
+        details: CallDetails;
+        headers: Record<string, string> | undefined;
+        remoteUserId: string;
+        state: CallState;
+        id: string;
+        direction: Direction;
+        incomingStream: MediaStream | undefined;
+        outgoingStream: MediaStream | undefined;
+        remoteUserDisplayName?: string;
+    }
+
+    enum CallState {
+        Init = 0,
+        Progressing = 1,
+        Established = 2,
+        Ended = 3,
+    }
+
+    enum Direction {
+        Inbound = 0,
+        Outbound = 1,
+    }
+
+    class CallDetails implements CallListener {
+        private session;
```

Comment:
```suggestion
```
you've private here and there. What's the purpose? Those won't be usable (and are not visible visible in linked docs)

Reply:
Thanks for pointing out.  I will remove those and update this PR 👍

Reply:
Fixed